src/components/PackButtons/index.jsx

```diff
@@ -9,11 +9,11 @@ const sharedSettings = {
 };
 
 const CreatePopup = () => (
-  <div className="create-popup">Test, yo!</div>
+  <div className="create-popup">Follow <a href="https://docs.stackstorm.com/reference/packs.html" target="_blank">Create and Contribute a Pack</a> guide. Once ready, submit a PR against PR against <a href="https://github.com/StackStorm-Exchange/exchange-incubator" target="_blank">exchange-incubator repo</a> and follow instructions there.</div>
 );
 
 const SubmitPopup = () => (
-  <div className="submit-popup">Yo momma is so fat she wouldn't even fit into our AWS pack.</div>
+  <div className="submit-popup">Nice form with automation is coming. Meantime, please submit your pack as a PR against <a href="https://github.com/StackStorm-Exchange/exchange-incubator" target="_blank">exchange-incubator repo</a>, and ping us on Slack or email <a href="mailto:support@stackstorm.com">support@stackstorm.com</a>.</div>
```

Comment:
I will removed "ping us" part since it's a bit redundant - we have notifications enabled and we get notified on Slack on #github when someone opens a new PR against that repo.